docs/SUMMARY.md

```diff
@@ -22,7 +22,7 @@
     * [Marketo](integrations/sources/marketo.md)
     * [MySQL](integrations/sources/mysql.md)
     * [MSSQL](integrations/sources/mssql.md)
-    * [Microsoft SQL Server (MSSQL)](integrations/sources/mssql.md)
+    * [Microsoft SQL Server \(MSSQL\)](integrations/sources/mssql-1.md)
```

Comment:
Wy are there two versions of the same doc here? `mssql.md` and `mssql-1.md`? SHoul

Reply:
Indeed! Fixed